CONTRIBUTING.md

```diff
@@ -21,25 +21,21 @@ Contributions following these guidelines are always welcomed, encouraged and app
 
 ## Project Release Process
 
-Jazzband aims to give full access to all members, including performing releases, as described in the
-[Jazzband Releases documentation](https://jazzband.co/about/releases).
+Releases require approval by a member of the `pip-tools-leads` team.
 
-To help keeping track of the releases and their changes, here's the current release process:
+This is the current release process:
 
-- Check to see if any recently merged PRs are missing from the milestone of the version about to be released.
-- Create a branch for the release. _Ex: release-3.4.0_.
-- Update the [CHANGELOG] with the version, date and add the text from [drafter release](https://github.com/jazzband/pip-tools/releases).
+- Create a branch for the release. _e.g., `release-3.4.0`_.
+- Use `towncrier` to update the [CHANGELOG], _e.g., `towncrier build --version v3.4.0`_.
 - Push the branch to your fork and create a pull request.
```

Comment:
Since this is talking about both upstream and fork. It's probably worth defining the assumed git remote names at the beginning of the document. Might be `origin` and `fork`, or `upstream` and `origin`.

Reply:
We have `upstream` + `origin` in `ansible/ansible`, I think. But I tend to use `fork`. Still, I know how to adapt so it doesn't matter.